Twilio-org/phonebank
# [Ticket 131] Fix Admin/Volunteer Ability to Edit Account Info Pages

Description: 
- Add check for is_admin and update Links based on admin status for:
  - Edit Account button on Account page
  - Cancel button on Edit Account page
  - Update button on Edit Account page
- Update account component snapshots and updateUser action tests to account for is_admin check

Test:
- run `npm run test:client users.test.js`
- Manually:
  - Login as an admin
  - Use navbar link to navigate to the Account page
  - Click on Edit Account 
  - Make sure url on Edit Account page is `/admin/account/<user_id>/edit`
  - Update something, then click the "Update" button
  - Make sure you are redirected to the Account page at `/admin/account/<user_id>`
  - Click on "Edit Account" button again
  - Click on "Cancel" button from the Edit Account page
  - Make sure you are redirected back to the Account page `/admin/account/<user_id>`
  - Repeat above steps on a volunteer account, but should be:
    - Edit Account: `/volunteers/account/<user_id>/edit`
    - Account: `/volunteers/account/<user_id>`
